src/lambda/commands/deploySamApplication.ts

```diff
@@ -26,19 +33,34 @@ export async function deploySamApplication(
 
     const tempFolder = await mkdtemp('samDeploy')
     const outputTemplatePath = path.join(tempFolder, 'template.yaml')
-
+    let stage = 'packaging'
     try {
         const packageInvocation = new SamCliPackageInvocation(template.fsPath, outputTemplatePath, s3Bucket, invoker)
+        restParams.outputChannel.show(true)
+        // TODO: Add nls support
+        restParams.outputChannel.appendLine(`Packaging SAM app to "${s3Bucket}" S3 bucket`)
```

Comment:
Minor: Consider: 
`Packaging SAM Application to S3 Bucket: ${s3Bucket}`

Reply:
✔